samples/wifi/radio_test/CMakeLists.txt

```diff
@@ -20,3 +20,36 @@ target_sources(app PRIVATE
 	src/nrf_wifi_radio_test_shell.c
 	src/nrf_wifi_radio_ficr_shell.c
 )
+
+
+if(SB_CONFIG_NETCORE_PERIPHERAL_RADIO_TEST)
```

Comment:
None of this will work, it needs to go in the sysbuild.cmake file, plus needs 2 space indentation for cmake

Reply:
> None of this will work, it needs to go in the sysbuild.cmake file, plus needs 2 space indentation for cmake

I did test this at least build wise and had seen hex files for both cores and merged files. My initial attempt was to use sysbuild.cmake but faced some errors and upon re-reading the doc link that you have shared above, for other processors child images only CMakeLists.txt was mentioned and sysbuild.cmake.